.github/scripts/ci-build.sh

```diff
@@ -27,6 +27,11 @@ fi
 plan_feature=${plan,,}
 moving_feature=${is_moving,,}
 
+
+# For some reason these cause issues with bindgen
```

Comment:
This is specific to the runner. We should move it to `ci.yml`.

Reply:
I'm not sure exactly how/what you mean. How do I clean up those files in `ci.yml`?